### PR TITLE
#CP-25 Setup React Project

### PR DESCRIPTION
**Pull request by samyshyaka**
#### What does this PR do?
Set up a React project from scratch

#### Description of Tasks to be completed

- Configure the project from scratch using web-pack and babel
- Setup React router and create a route for the landing page
- Add a ReadMe file which includes installation and run commands

#### How should this be manually tested?
Clone the project repository linked [here](https://github.com/atlp-rwanda/phantom-fe-codebandits)

```
cd phantom-fe-codebandits/
git checkout ch-setup-react-project-cp-25
git fetch origin
npm install
npm run start
run http://localhost:3000 in your browser
```

#### What are the relevant Trello Cards?
[#CP-25](https://trello.com/c/PrsqDuPS)

### CODE REVIEW

**Review by feyton — APPROVED** [review]
These look good.

And the project is learning successfully

**Review by Yvad60 — APPROVED** [review]
Good job, it's working fine!
is it on my end or the server should start automatically in the browser?

**Review by Abdulkeza — APPROVED** [review]
Great Job sam! on my side it is running.

**Review by Jkadhuwa — CHANGES_REQUESTED** [review]
Nice work, try to leave an empty line after the last line of code.

**Comment by samyshyaka** [comment]
> 

Just did that

**Review by Jkadhuwa — APPROVED** [review]

**Comment by samyshyaka** [comment]
> Good job, it's working fine! is it on my end or the server should start automatically in the browser?

It starts automatically and if it doesn't, you just use the link

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlp-rwanda/phantom-fe-codebandits.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atlp-rwanda/phantom-fe-codebandits/issues"
+  },
+  "homepage": "https://github.com/atlp-rwanda/phantom-fe-codebandits#readme",
```
Comment (feyton) [inline package.json]: @samuel-shyaka-dus This line can create conflict when deploying to GitHub pages. The GitHub Pages package look for this entry **(homepage)** to associate the corresponding link to use and would be formatted as **[http://atlp-rwanda.github.io/phantom-fe-codebandits](http://atlp-rwanda.github.io/phantom-fe-codebandits)**

[Documentation linked here](https://www.c-sharpcorner.com/article/how-to-deploy-react-application-on-github-pages/#:~:text=add%20homepage%20property%20to%20package.json%20file)

**Review by feyton — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (samyshyaka) [reply package.json]: Just changed that

**Pull request merged**